src/components/setup/addon.module.tsx

```diff
@@ -0,0 +1,42 @@
+import TagComponent from "@components/global/tag.module";
+import { faker } from "@faker-js/faker";
+import { Addon } from "@prisma/client";
+import React, { useMemo } from "react";
+import { BsWifi, BsWifiOff } from "react-icons/bs";
+import styles from "./addon.module.scss";
+
+// TODO display correct card count.
+
+function AddonComponent({ addon, active, onclick }: Props) {
+  const avatar = useMemo(() => faker.image.abstract(640, 640, true), []);
+  return (
+    <article className={styles.main} onClick={onclick}>
+      {/* Addon avatar */}
+      <img className={styles.avatar} src={avatar} alt="a" />
+
+      {/* Title, card count and tags */}
+      <section className={styles.info}>
+
+        <h1>{addon.title}</h1>
```

Comment:
Semantically you only want one `h1` per page, since you're mapping over multiple addons that will go wonky here.

I'd use a simple `<p>` and increase its font size / weight with css instead